www/assessment/distractor-rationale.php

```diff
@@ -108,16 +108,15 @@
             response</em> allows Authors to write unique feedback against each possible response.
         </p>
         <p>
-            In Assessments and Reports, Learnosity provides two approaches to display distractor rationale
-            for multiple choice questions:
+            Learnosity provides two approaches to display distractor rationale:
         </p>
         <ol>
             <li><p>Initialization options</p></li>
             <li><p>JavaScript method</p></li>
         </ol>
         <p>
-            For questions other than <abbr title="Multiple Choice Question">MCQ</abbr>, or if you would like to implement further hints and feedback, you can write
-            custom code to display Distractor Rationale and other metadata.
+            Currently, we provide out of the box support for Mulitple choice, Cloze with drop down, Cloze with drag & drop, Cloze with text.
```

Comment:
Minor one here @tomas-bocanera-lrn:
There's a typo on line 118: "`Mulitple` choice" instead of `Multiple`

Reply:
Thanks @hbalseiro . Fixed. FYI @Ninavdk